metadataStore/commands.py

```diff
@@ -370,6 +370,31 @@ def find_last():
     return Header.objects.order_by('-_id')[0:1][0]
 
 
+def search_events_broker(beamline_id, start_time, end_time):
```

Comment:
Because Dan asked for this specifically for the broker. We had a convo this afternoon. It will go sleepy bye tmrw latest!=